### DISCUSSION TITLE
Set color for bottom bar

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#7148] [Q&A]
![-2147483648_-210242](https://user-images.githubusercontent.com/69314237/134040419-7b294996-6581-4031-ab39-e6ab473fdd9a.jpg)
The color of my app is dark, but the color of the bottom bar is white. I want to change the color of this panel, but I can't figure out how.

**Comment by jeromelaban** [comment]
Can you provide the XAML of the page that shows gives you this result ?
